index.html

```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
                 <div class="lets-connect">
                     <h5>LET'S CONNECT</h5>
                 </div>
-
                 <div class="icons">
                     <ul>
                         <li><a href="https://twitter.com"><img src="Pictures/twitter.png" alt=""></a></li>
-                        <li><a href="https://www.linkedin.com/in/giorgi-bokuchava-430252240/"><img src="Pictures/Linkedin.png" alt=""></a></li>
+                        <li><a href="https://www.linkedin.com/in/giorgi-bokuchava-430252240/"><img
+                                    src="Pictures/Linkedin.png" alt=""></a></li>
                         <li><a href="https://medium.com"><img src="Pictures/medium.png" alt=""></a></li>
                         <li><a href="https://github.com/Boku52500"><img src="Pictures/github.png" alt=""></a></li>
                         <li><a href="https://angellist.com"><img src="Pictures/victory.png" alt=""></a></li>
                     </ul>
                 </div>
             </div>
-        </section>
+    </section>
+
+
+
+
+    <section class="2-page">
+        <div class="snapshot">
+            <a href=""><img src="Pictures/snapshot.png" alt=""></a>
```

Comment:
- [ ] Kindly remove the anchor tag `a` around the project screenshot because it's just an image not a link 👍